.github/workflows/main.yml

```diff
@@ -223,6 +224,31 @@ jobs:
           ./simple-transfer-${GITHUB_RUN_NUMBER}/simple-transfer dispose
           aws s3 rm s3://ziti-smoketest-fablab-instances/simple-transfer-${GITHUB_RUN_NUMBER}.tar.gz.gpg
 
+  quickstart:
+    name: Run the Go Quickstart
+    # - always() allows evaluating further conditional expressions even if
+    #   needed jobs were skipped
+    if: |
+      always()
+    runs-on: ubuntu-20.04
+    needs: [ linux-build ]
+    steps:
+      - name: Git Shallow Checkout
+        uses: actions/checkout@v3
+
+      - name: Download linux release artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: linux-release-${{ github.run_id }}
+          path: release/
+
+      - name: Run Go Quickstart
+        run: |
+          for ZITI in ./release/amd64/linux/ziti; do 
+            chmod +x "$ZITI"
+            "$ZITI" edge quickstart
```

Comment:
this CI job will never complete. This command runs a controller and router in the foreground. It'll never exit until you ctrl-c (or exit etc). I don't think this test, as it is written, should be in the path.

Reply:
I'll adust this branch to reflect what I learn about the test as it's being developed. I hadn't successfully run it locally yet, so don't yet understand the rationale of the quickstart command. 

Now it sounds like it's intended to run the controller and possibly a router in the foreground, waiting for input.

In that case, we probably need two processes: the quickstart in the background with a readiness check, and the quickstart_test in the foreground.